worker/src/main.rs

```diff
@@ -361,12 +360,18 @@ fn worker(
         println!("[<] Extrinsic got finalized. Hash: {:?}\n", tx_hash);
     }
 
+    crate::db_handler::DBHandler::load_from_disk().expect("Failed to load data from disk");
+
```

Comment:
This could panic and the lines above are infested with `unwrap()` What are the consequences of a panic in this function

Reply:
The worker would shut down right at start up. I guess this is okay, since with a worker not being able load data from disk there is something definitely wrong. Though of course it should not panic when the disk is empty.

Reply:
It only panics if there is data on disk but it fails to load it.